appinterface/projection/crossfire/httpclient.go

```diff
@@ -47,7 +46,7 @@ func (client *HTTPClient) Participants() (*[]Participant, error) {
 func (client *HTTPClient) request(method string, queryString ...string) (io.ReadCloser, error) {
 	var err error
 
-	url := client.url
+	url := client.url + "/" + method
```

Comment:
I think there is no need to add `method` here, since it's a JSON file.

Reply:
Yes, I agree but due to https://github.com/crypto-com/chain-indexing/pull/264#discussion_r553123893 I had to do this.